doc/content/integrations/node-red/send/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Send Messages"
+description: ""
+weight: 3
+---
+
+This section follows the process of setting up a flow which publishes messages to a certain topic that the MQTT Server is subscribed to.
+
+Doing this schedules downlink messages to be sent to your end device. This section follows the example for publishing downlink traffic in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide.
+
+## Configure MQTT Out Node
+
+Find the **mqtt out** node in the Node-RED library and place it on the dashboard. This node connects to the MQTT broker and publishes messages. Configure the **Server** options with the same settings as used in the [Receive Events and Messages]({{< ref "/integrations/node-red#receive-events-and-messages" >}}) section before.
+
+As stated in [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide, topic `v3/{application_id}/devices/{device_id}/down/push` can be used for scheduling downlink messages. Choose a **QoS** from listed options and state whether you want the MQTT Server to retain messages. 
+
+{{< figure src="mqtt_out_node_properties.png" alt="mqtt out node properties" >}}
+
+Further, find and place the **inject** node on the dashboard. This node injects a message into a flow and can be triggered manually, but it is also possible to define regular intervals between the automatic injections. Double-click on the **inject** node to configure its properties. The **Payload** field can have different formats - in this example, JSON is used. Expand the **Payload** field and enter this:
+
+```json
+{
+  "downlinks": [{
+    "f_port": 15,
+    "frm_payload": "vu8=",
+    "priority": "NORMAL"
+  }]
+}
+```
+
```

Comment:
I would split this in two pieces. It's pretty hard to get the right base64 payload manually in here.

The two pieces would be the **inject** node that creates a buffer, and drafting the JSON payload to be published.

Here's an example:

```json
[{"id":"9602047.883b9f8","type":"inject","z":"787d143a.05e53c","name":"","topic":"","payload":"[0, 42, 255, 0]","payloadType":"bin","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":190,"y":380,"wires":[["b691b63f.189828"]]},{"id":"5ec92ae4.08b554","type":"debug","z":"787d143a.05e53c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":690,"y":380,"wires":[]},{"id":"b691b63f.189828","type":"function","z":"787d143a.05e53c","name":"convert to downlink message","func":"return {\n  payload: {\n    downlinks: [\n        {\n            f_port: 15,\n            frm_payload: msg.payload.toString('base64'),\n            priority: \"NORMAL\"\n        }\n    ]\n  }\n}","outputs":1,"noerr":0,"x":430,"y":380,"wires":[["5ec92ae4.08b554"]]}]
```

(use menu -> **Import** to import this)

You basically get this:

<img width="735" alt="Screenshot 2020-06-03 at 11 57 59" src="https://user-images.githubusercontent.com/13334001/83623332-7d584800-a591-11ea-8fc2-5964ad30fffe.png">